src/server/services/validate-upload.js

```diff
@@ -200,8 +200,7 @@ async function validateSubrecipientRecord ({ upload, record: recipient, typeRule
         uei: recipient.Unique_Entity_Identifier__c,
         tin: recipient.EIN__c,
         record: recipient,
-        upload_id: upload.id,
-        tenant_id: upload.tenant_id
```

Comment:
I think I'm comfortable taking that risk for now.  I guess the situation where this could get messy is if some future change adds support for some sort of super administrator (multi-tenant) view.